e2e-version/index.js

```diff
@@ -49,11 +49,12 @@ async function run() {
         }
     }
 
-    if (VersionResolverTypes.PluginGrafanaDependency) {
+    if (versionResolverType === VersionResolverTypes.PluginGrafanaDependency) {
       // limit the number of versions to 6
       output = evenlyPickVersions(output, VERSIONS_LIMIT);
     }
 
+    console.log('Resolved versions: ', output);
```

Comment:
is this console log intended here?

Reply:
Yeah this will print the output in the GH action console which can be useful. It seems like the @actions/core package doesn't have a `debug` logger, so console log it what [people is suggesting](https://github.com/actions/toolkit/issues/108).